language/move-core/types/src/language_storage.rs

```diff
@@ -41,8 +42,8 @@ pub struct StructTag {
     pub address: AccountAddress,
     pub module: Identifier,
     pub name: Identifier,
-    // TODO: rename to "type_args" (or better "ty_args"?)
-    #[serde(rename(serialize = "type_args", deserialize = "type_args"))]
+    // alias for compatibility with old json serialized data.
```

Comment:
Can you add back the TODO?

Reply:
The reason this changed was we use **serde-reflection** to generate rust code from the YAML file generated from **serde**. If the YAML file use lowercaswe, serde-reflection will fail since the lowcase is reserved for keywords. We previously though we could just alias or deserialized to serialize to lowercase while deserialized back to uppercase. But it doesn't work as expected since serde-reflection doesn't  know how serde want to deserialize. We just endup hack our serde-reflection to make the rust generation work. 

In terms of @jolestar's change, It lgtm and it shouldn't break our serde-reflection as well since the generated YAML file doesn't change.

Reply:
@tnowacki Add the TODO back. I mistakenly thought adding `serde(rename)` finished the renaming.